AmplifyCombineSupport/AmplifyCombineSupport/DataStore.swift

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+import Combine
+
+public typealias DataStorePublisher<Output> = AnyPublisher<Output, DataStoreError>
+
+public extension DataStoreBaseBehavior {
```

Comment:
Will investigate to see if it's possible to automatically import this as part of `import Amplify; import Combine` as long as the support library is installed. Considerations:

1. Can the solution apply to non-CocoaPods installations (e.g., SPM, Carthage, framework)?
2. Does the solution feel not-too-surprising? E.g., I would rather be explicit and require an `import AmplifyCombineSupport` until we officially roll it into Amplify Core in a couple of years, than make customers wonder why they have to `import Combine` to use the Combine-flavored APIs, even though they’re using any core Combine APIs in that file. Realistically this might not be a problem, since you'll often have to store a reference to an `AnyCancellable` or `Publisher` of some kind, which requires a Combine import.